### PR TITLE
feat(permissions): Update

### PR DESCRIPTION
**Pull request by jfagoagas**
### Description

Update Prowler permissions with the latest IAM actions required.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: iam/create_role_to_assume_cfn.yaml [hunk 1]
```diff
@@ -56,20 +56,27 @@ Resources:
             - Effect: Allow
               Action:
                 - 'account:Get*'
-                - 'appstream:DescribeFleets'
+                - 'appstream:Describe*'
+                - 'codeartifact:List*'
+                - 'codebuild:Batch*'
```
Comment (MrCloudSec) [inline iam/create_role_to_assume_cfn.yaml]:
```suggestion
                - 'codebuild:BatchGet*'
```

Comment (MrCloudSec) [reply iam/create_role_to_assume_cfn.yaml]: Prowler only needs BatchGet permissions in CodeBuild.

**Review by MrCloudSec — COMMENTED** [review]
Review my comment pls.

File: iam/prowler-additions-policy.json [hunk 2]
```diff
@@ -4,15 +4,16 @@
         {
             "Action": [
                 "account:Get*",
-                "appstream:DescribeFleets",
-                "codeartifact:ListRepositories",
-                "codebuild:BatchGetBuilds",
+                "appstream:Describe*",
+                "codeartifact:List*",
+                "codebuild:Batch*",
```
Comment (jfagoagas) [inline iam/prowler-additions-policy.json]:
```suggestion
                "codebuild:BatchGet*",
```

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**